cloudformation/imagebuilder/custom_ami_build_component.yml

```diff
@@ -0,0 +1,231 @@
+name: PCluster
```

Comment:
Should it be yaml instead of yml?

Reply:
Yes, it should be yaml. Replace `custom_ami_build_component.yml` and `official_ami_build_component.yml` with `update_and_reboot.yaml` and `pcluster_install.yaml` to isolate internal and external build.